.github/workflows/test.yml

```
@@ -10,18 +10,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tarantool: ['1.10', '2.5', '2.6', '2.7', '2.8']
+        tarantool: ['1.10', '2.5', '2.6', '2.7', '2.8', '3.2']
```

Let's remove the outdated versions:
```suggestion
        tarantool: ['1.10', '2.10', '2.11', '3.1', '3.2']
```

---

Don't we want to remove 1.10 completely then?

---

This is a debatable issue, but it seems to me that there should be testing on branch 1.x until it breaks. This way we will know that we are still working on 1.x.